packages/cypress/package.json

```diff
@@ -47,6 +47,8 @@
     "typescript": "^5.5.2"
   },
   "dependencies": {
+    "@sentry/node": "^8.13.0",
+    "@sentry/profiling-node": "^8.13.0",
```

Comment:
ideally, we'd be using the same version range everywhere but right now we are using a mix of `^8.13.0` and `^8.15.0` in this PR